lib/ui/painting.dart

```diff
@@ -4125,7 +4127,7 @@ class Picture extends NativeFieldWrapperClass2 {
   ///
   /// Although the image is returned synchronously, the picture is actually
   /// rasterized the first time the image is drawn and then cached.
-  Future<Image> toImage(int width, int height) {
+  Future<Image> toImage(int/*!*/ width, int/*!*/ height) {
```

Comment:
Does the return type here or the types in `_toImage` need annotations?

Reply:
No, but I'll annotate anyway. I try to annotate everything that's public and manually reviewed.